src/app/components/normal-component/verify-account/verify-account.component.ts

```diff
@@ -39,9 +39,9 @@ export class VerifyAccountComponent implements OnInit, OnDestroy {
     public async ngOnInit(): Promise<void> {
         this.userSub = this.authService.getUserObs()
             .subscribe(async(user: AuthUser) => {
-                this.emailAddress = user.email;
+                this.emailAddress = user.email.get();
                 // We know that if this page is shown, something needs to be done to finalize the account
-                if (user.username == null || user.username === '') {
+                if (user.username.isAbsent() || user.username.get() === '') { // TODO: get rid of all ''
```

Comment:
what about this new TODO ?

Reply:
TODO qui a été fait, les '' utilisés pour du null ont été enlevés (TODO enlevé)

Reply:
Mais du coup là on continue de considerer '' comme valeur acceptable d'username, or '' représentait "absent" donc maintenant que null est bloqué par strictness, absent devrais suffir et '' être bloqué en amont, c'est bloquer en amont le but du TODO idéalement non ?
c'est plutôt fait non ? (si oui, username.isAbsent() est une condition suffisante)

Reply:
ah oui, ce check est inutile car un username peut pas être '' (sauf si on le change manuellement dans firebase ofc)